src/components/RadioButton/RadioButton.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import bem from 'bem';
+import Text from '../Text';
+import styles from './RadioButton.scss';
+import { CHECKBOX_VIEWBOX } from '../../constants';
+
+const { block, elem } = bem({
+    name: 'RadioButton',
+    classnames: styles,
+    propsToMods: ['disabled']
+});
+
+const RadioButton = props => {
+    const { id, children, viewbox, disabled, ...rest } = props;
+
+    return (
+        <div {...block(props)}>
+            <input type="radio" id={id} {...rest} {...elem('input', props)} disabled={disabled} />
+            <label {...elem('label', props)} htmlFor={id} disabled={disabled}>
+                <span {...elem('box', props)}>
+                    <svg
+                        {...elem('svg', props)}
+                        width="12px"
+                        height="10px"
+                        viewBox={CHECKBOX_VIEWBOX}
+                    >
+                        <circle cx="5" cy="6" r="3" />
+                    </svg>
+                </span>
+                <Text {...elem('text', props)} inline muted={disabled}>
```

Comment:
We should probably only render this if `children` is not `null`, e.g.:

```
{ !!children && (
   <Text ... />
)}
```

(Same for Checkbox component, now that I think of it)

Reply:
children isRequired via props. Otherwise you have no text at all, only a button/checkbox.

Reply:
At this point I don't exactly remember the use case I had in mind but I think you should be able to render just a checkbox / radio, without requiring it to come with a label

As an example, consider this use case in the maintenance ui:
![screenshot 2019-01-28 at 17 05 00](https://user-images.githubusercontent.com/798175/51848925-e46f4800-231e-11e9-965c-037da8e67b4d.png)

Reply:
would allowing for children to be `node` or `string` but still required make sense. Then if it is a string, we make it a `Text` as now, or render it without change if it is a node.

Reply:
I'm still not sure why any children would be required. Native HTML radios don't require any label  either: http://jsfiddle.net/7joxtrzn/

Reply:
I think it is strange to allow for that. But whatever